rasa/core/processor.py

```diff
@@ -742,6 +745,9 @@ async def _run_prediction_loop(
                     self.on_circuit_break(tracker, output_channel, self.nlg)
                 break
 
+            if prediction.is_end_to_end_prediction:
+                tracker = await self._run_action_extract_slots(output_channel, tracker)
```

Comment:
can we please add a log statement to make it clear that we're running this a second time?

Reply:
Should I also add a changelog entry for this - or do you think the global slot mappings initial changelog encapsulates this too?

Reply:
As we haven't released yet the initial changelog encapsulates this 👍🏻